README.md

```diff
@@ -42,6 +42,7 @@ Help welcome, eg by filling in the entries with `?` `TODO` and `CHECKME`, correc
 | User defined operators | partial: opCall opSlice, opAssign etc | yes | -1 |
 | User defined attributes | yes | ? | ? |
 | RAII | yes (modulo caveats https://github.com/timotheecour/D_vs_nim/issues/27) | no, see: [RAII](https://forum.nim-lang.org/t/362/1) | 1 |
```

Comment:
> However, tables (which are like D’s maps)  [...]

just say that only pointer/ptr/ref types (and builtin type `cstring` type) can be nil. `tables.Table` is an object so cannot be nil (and comparison to nil would be CT error)

Reply:
Updated text:

> Lessening the impact of this, many built-in types can never be `nil`, including `seq`s. Only `ptr` and `ref` types (and `cstring`s) can be `nil`.